scripts/baseline_comparison/evaluate_utils.py

```diff
@@ -27,11 +27,15 @@
 class Experiment:
     expname: str  # name of the parent experiment used to store the file
     name: str  # name of the specific experiment, e.g. "localsearch"
-    run_fun: Callable[[], List[ResultRow]]  # function to execute to obtain results
+    run_fun: Callable[..., List[ResultRow]]  # function to execute to obtain results
```

Comment:
`Any` or `object` is probably better than the ellipsis as ellipsis is a type that wont match here right?